src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+object Calculator {
+    private const val FIRST_OPERAND_INDEX = 0
+    private const val FIRST_OPERATOR_INDEX = 1
+    private const val OPERATION_UNIT = 2
```

Comment:
코틀린 코드 컨벤션에 따르면 [상수 선언](https://kotlinlang.org/docs/coding-conventions.html#property-names)의 경우 `top-level property`로 지정해보면 어떨까요? 

```suggestion
private const val FIRST_OPERAND_INDEX = 0
private const val FIRST_OPERATOR_INDEX = 1
private const val OPERATION_UNIT = 2
    
object Calculator {
```

Reply:
첨부해주신 링크의 네이밍 규칙에 의하면
불변값을 담은 object 의 val 프로퍼티도 SCREAMING_SNAKE_CASE 로 쓸 수 있는 것 같은데요.
그래서 현재처럼 사용해도 무방하다고 생각이 듭니다.
어떤 점 때문에 top-level 로 옮겼으면 좋겠다고 생각하시는지 질문드립니다

또한 해당 상수들이 Calculator 라는 객체와만 관련있는 것 같아서 안에다 명시하면 좋을 것 같다는 생각인데 어떻게 생각하시는지 궁금합니다.

Reply:
제가 `top-level object val properties`라는 표현에서 **object val properties**의 의미를 조금 잘못 이해했던 것 같아요. 🤔
object 의 val property가 지금 성호님이 작성하신 코드의 형태와 동일했군요? (사실 object 잘 안쓰는편이라 생각보다 낯서네요😅) 

> Calculator 라는 객체와만 관련있는 것 같아서 안에다 명시하면 좋을 것 같다는 생각인데 어떻게 생각하시는지 궁금합니다.

위와 같은 생각을 가지신거라면 저는 `companion object` block 안에서, const val(const를 쓸 수 없는 자료형이라면 그냥 val)로 상수를 선언하여, `Calculator.상수명` 이런 형태로 쓰고있어요!  

성호님과 이렇게 소통하다보니 사실상 `object의 val 프로퍼티`랑 `companion object val 프로퍼티(?)`랑 모양새도 그렇고 쓰임새도 그렇고 사실상 거의 동일하다는 생각이드네요. 😃